js/web/pages/profiles/index.js

```diff
@@ -14,6 +15,6 @@ export default function Profiles() {
       path = '/'
     }
     router.push(path)
-  }, [wallet])
+  }, [wallet, router])
```

Comment:
think this may be a straggler as well, plz dbl check everything is straigth after removing